src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesSlave.java

```diff
@@ -39,25 +32,23 @@
     private final KubernetesCloud cloud;
 
     @DataBoundConstructor
-    public KubernetesSlave(DockerTemplate template, String nodeDescription, KubernetesCloud cloud, Label label)
+    public KubernetesSlave(PodTemplate template, String nodeDescription, KubernetesCloud cloud, Label label)
```

Comment:
@tfennelly The plugin has only two release, it is a *young plugin*. If we are very strict with that, in one year we'll have more deprecated code than functional code :)